src/Data/Uniques/race.lua

```diff
@@ -0,0 +1,151 @@
+-- Item data (c) Grinding Gear Games
+
+return {
+-- Weapon: One Handed Axe
+[[
+Ashes of the Sun
+Driftwood Club
+League: Endless Races
+Quality: 0
+Sockets: W-W-W
+LevelReq: 1
+Implicits: 1
+10% Reduced Enemy Stun Threshold
+Adds 3 to 6 Fire Damage
+20% Increased Attack Speed
+500% Increased Ignite Duration on Enemies
+25% Increased Burning Damage
+30% Chance to Ignite
+]],[[
+Blood of Summer
+Rusted Sword
+League: Endless Races
+Quality: 0
+Sockets: W-W-W
+Implicits: 1
+{tags:attack}40% increased Global Accuracy Rating
```

Comment:
`{tags:...}` only apply to Catalyst modifiable stats which only include: rings, belts, amulets.

Reply:
Done